test-cases/performance/perf-regression-throughput-1250gb-with-ebs-volume.yaml

```diff
@@ -0,0 +1,39 @@
+test_duration: 1580
+
+prepare_write_cmd: ["cassandra-stress write no-warmup cl=ALL n=312500000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..312500000",
+                    "cassandra-stress write no-warmup cl=ALL n=312500000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=312500001..625000000",
+                    "cassandra-stress write no-warmup cl=ALL n=312500000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=625000001..937500000",
+                    "cassandra-stress write no-warmup cl=ALL n=312500000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=937500001..1250000000"]
+
+
+stress_cmd_w: "cassandra-stress write no-warmup cl=QUORUM n=1250000000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=100 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..1250000000"
+stress_cmd_r: "cassandra-stress read no-warmup  cl=QUORUM duration=60m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate 'threads=90' -col 'size=FIXED(1024) n=FIXED(1)' -pop 'dist=gauss(1..1250000000,625000000,1250000000)' "
+stress_cmd_m: "cassandra-stress mixed no-warmup cl=QUORUM duration=60m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate 'threads=90' -col 'size=FIXED(1024) n=FIXED(1)' -pop 'dist=gauss(1..1250000000,625000000,1250000000)' "
+
+n_db_nodes: 3
+n_loaders: 4
+n_monitor_nodes: 1
+
+instance_type_loader: 'c5.2xlarge'
+instance_type_monitor: 't3.large'
+instance_type_db: 'r5b.2xlarge'
+
+user_prefix: 'perf-ebs-throughput-1250'
+space_node_threshold: 644245094
+ami_id_db_scylla_desc: 'VERSION_DESC'
+
+round_robin: true
+append_scylla_args: '--blocked-reactor-notify-ms 5'
+backtrace_decoding: false
+
+use_mgmt: true
+
+store_perf_results: true
+send_email: true
+email_recipients: ['scylla-perf-results@scylladb.com']
+backtrace_decoding: false
+
+data_volume_disk_num: 5
+data_volume_disk_type: "gp2"
+data_volume_disk_size: 500
+data_volume_disk_iops: 10000 # depend on type iops could be 100-16000 for io2|io3 and 3000-16000 for gp3
```

Comment:
Same as above.

Reply:
This shouldn't be here.
But let's remove throughput completely for now.

Reply:
Done